time_tracker_api/time_entries/time_entries_model.py

```diff
@@ -106,6 +111,24 @@ def create_sql_date_range_filter(date_range: dict) -> str:
         else:
             return ''
 
+    def find(
+        self,
+        id: str,
+        event_context: EventContext,
+        peeker: 'function' = None,
```

Comment:
`'function'` what kind of data is this?

Reply:
Is a type hinting supposed to represent that this parameter receives a function. Is part of a previous implementation. Looking on internet, It seems the right type hint is callable -> https://stackoverflow.com/questions/45897225/what-is-the-type-hint-for-a-function I guess the current notation was preferred over Callable, because it looks more expressive about the type of the parameter.

Reply:
So it's ok to change this for a better design.  I opened a ticket for it https://github.com/ioet/time-tracker-backend/issues/135